bin/check-rds.rb

```diff
@@ -102,6 +102,25 @@ class CheckRDS < Sensu::Plugin::Check::CLI
          description: 'Continue if CloudWatch provides no metrics for the time period',
          default: false
 
+  option :use_iam_role,
```

Comment:
if credentials are not supplied the sdk will default to use the iam instance profile.